app/views/users/show.html.erb

```diff
@@ -0,0 +1,34 @@
+<h1><%= @user.username %></h1>
+
+<h2>Own photos</h2>
+
+<% @user.own_photos.each do |photo| %>
+  <div class="row mb-4">
+    <div class="col-md-6 offset-md-3">
+      <div class="card">
+        <%= image_tag photo.image, class: "card-img-top" %>
+        <div class="card-body">
+          <h5 class="card-title"><%= photo.owner.username %></h5>
+          <p class="card-text"><%= photo.caption %></p>
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
+                </div>
+                <div class="flex-grow-1 ms-3">
+                  <h5 class="mt-0"><%= comment.author.username %></h5>
+                  <p><%= comment.body %></p>
+                </div>
+              </div>
+            </li>
+          <% end %>
+        </ul>
+        <div class="card-body">
+          <%= render partial: "comments/form", locals: { comment: Comment.new(photo_id: photo.id) } %>
```

Comment:
This is good! There is another way of doing this such as `<%= render "comments/form", comment: Comment.new(photo_id: photo.id) %>`. The way you did it is correct, just makes it more explicit, but if you're looking for a shorter version the one i provided also works.